main.rb

```diff
@@ -1,6 +1,20 @@
 require_relative 'app'
+require 'fileutils'
+
 def main()
   app = App.new
   app.run
 end
```

Comment:
I'm receiving an error when I run the `ruby main.rb`  on the terminal here's a screenshot of it 

![image](https://user-images.githubusercontent.com/103601335/218629586-31e48a0f-84f5-470e-92c4-a217939766ab.png)